Cargo.toml

```diff
@@ -32,7 +32,7 @@ default-members = ["base", "integer", "float", "rational", "macros"]
 # TODO(next): Assign a version name to all crates that haven't reached v1. For crates that already reaches v1, we assume that the update frequency of that crate is low, so it's unnecessary to create a version tag until v2.0 is released
 #             Our strategy for pre v1 dependencies: the bump of default version is not considered a break change, only minor version is required to bump.
 #             Our strategy for v1 dependencies: the bump of default version is considered a break change, the support for new version will be release through a versioned feature flag, and the default version will be bumped in the next major version.
-default = ["std", "num-order"]
+default = ["std", "num-order", "num-traits", "num-traits_v02"]

```

Comment:
Unfortunately I will not accept adding `num-traits` as a default dependency. Please specify the dependency in your own Cargo.toml